src/app/pages/session-detail/session-detail.page.ts

```diff
@@ -131,8 +136,13 @@ export class SessionDetailPage implements OnInit {
     }
   }
 
-  share() {
-    // ToDO implement share feature
+  async share() {
+    let sharableLink = await this.sessionService.getShareSessionId(this.id);
+    let url = CommonRoutes.SESSIONS+"/"+CommonRoutes.SESSIONS_DETAILS+"/"+sharableLink.shareLink;
+    let link = await this.utilService.getDeepLink(url);
+    console.log(link);
+    let params = {link: link, subject: this.sessionHeaderData?.title, text: "Join this session using the link provided here "}
```

Comment:
The message is not coming in the API response?

Reply:
While link generation? No